content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -31,6 +32,8 @@ The CertificateSigningRequest object includes a PEM-encoded PKCS#10 signing requ
 the `spec.request` field. The CertificateSigningRequest denotes the signer (the
 recipient that the request is being made to) using the `spec.signerName` field.
 Note that `spec.signerName` is a required key after API version `certificates.k8s.io/v1`.
+The client may optionally set the `spec.durationHint` to request a particular
+lifetime for the issued certificate.
```

Comment:
For this first mention of the field:
- I'd mention it's only available in Kubernetes v1.22 and later
- I'd also tell people what the type of the field is. They might for example think it takes a number of seconds or a number of days, or more likely be frustrated because both could be valid and they're not sure.

Reply:
yes to noting the version this is supported in

API convention is to include the unit in the field name and make it an integer, not a string (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units)

Reply:
I guess I'd like this to be called `durationSecondsHint` or similar, then (and be an integer).

Reply:
If the field name is already decided, then instead we must document what units it uses. Ideally the API reference for CertificateSigningRequest will use the same, clear working.

Reply:
final field name is expirationSeconds